src/Features/Core/Portable/InheritanceMargin/IRemoteInheritanceMarginService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.InheritanceMargin
+{
+    internal interface IRemoteInheritanceMarginService
+    {
+        ValueTask<ImmutableArray<SerializableInheritanceMarginItem>> GetInheritanceMarginItemsAsync(
+            PinnedSolutionInfo pinnedSolutionInfo,
+            ProjectId projectId,
+            ImmutableArray<(SymbolKey symbolKey, int lineNumber)> symbolKeyAndLineNumbers,
```

Comment:
this feels a bit odd.  why do you need to pass the line number in...

Reply:
oh.  i see.  it's so when you get the answers back, you can associate them with teh item you originally collected and called into this with.  it's def not that clean.  but it's not awful.  it's more like an 'id' in this regard rather than a line-number.

Reply:
: ) Yes, the scenario is when a set of symbols is sent to the OOP, those without inheritance target would not be returned.
The line number info is collected in devenv with the symbol, if not passing that to the OOP, then it needs an 'ID' in the return result to match the inheritance targets to the line number. 
I did this just because imo creating a 'int' ID just for an int line number is also wired. So I choose just pass it in

Reply:
yup.  that makes sense.  might be worthwhile to doc.